WooCommerce/WooCommerceTests/ViewRelated/Orders/Order Details/Addresses/CountrySelector/CountrySelectorViewModelTests.swift

```diff
@@ -5,57 +5,69 @@ import TestKit
 
 final class CountrySelectorViewModelTests: XCTestCase {
 
+    let sampleSiteID: Int64 = 123
+    let testingStorage = MockStorageManager()
+
+    override func setUp () {
+        super.setUp()
+
+        testingStorage.reset()
+        testingStorage.insertSampleCountries(readOnlyCountries: Self.sampleCountries)
+    }
+
     func test_filter_countries_return_expected_results() {
         // Given
-        let viewModel = CountrySelectorViewModel()
+        let viewModel = CountrySelectorViewModel(siteID: sampleSiteID, storageManager: testingStorage)
 
         // When
         viewModel.searchTerm = "Co"
         let countries = viewModel.command.data.map { $0.name }
+
         // Then
         assertEqual(countries, [
             "Cocos (Keeling) Islands",
-            "Congo - Kinshasa",
+            "Colombia",
+            "Comoros",
             "Congo - Brazzaville",
+            "Congo - Kinshasa",
             "Cook Islands",
-            "Colombia",
             "Costa Rica",
-            "Comoros",
-            "Morocco",
-            "Monaco",
             "Mexico",
+            "Monaco",
+            "Morocco",
```

Comment:
The order of these changed because the `ResultsController` now filters them by name.